### ISSUE TITLE
Improve Speed of Board Generation

### ISSUE THREAD

**Issue by anirudhRowjee** [anirudhRowjee/py-sudoku#2]
Implement a well-known sudoku generation Algorithm to generate the board as opposed to random number generation.

**Comment by thevectorguy** [comment]
Can i work on this?

**Comment by CoderCatA5** [comment]
can i work on this

**Comment by thecoderash** [comment]
yes y'all @CoderCatA5 @SanjanaSogimatt @prishagoel can work on this

**Comment by ArnavKumar7** [comment]
yeah , for sure :) all the best

**Comment by saisudhirs** [comment]
Hi, can I work on this issue.

**Comment by Smuzzy-waiii** [comment]
Hey so i must add a new file which will print sudoku, implementing a famous sudoku algorithm
Is that correct?

**Comment by saisudhirs** [comment]
@ArnavKumar7 Please assign me to this issue at the earliest as only an hour of time is left.

**Comment by thecoderash** [comment]
@saisudhirs yes u can work on this
